src/seedu/addressbook/AddressBook.java

```diff
@@ -558,6 +565,39 @@ private static String getMessageForSuccessfulDelete(String[] deletedPerson) {
         return String.format(MESSAGE_DELETE_PERSON_SUCCESS, getMessageForFormattedPersonData(deletedPerson));
     }
 
+    /**
+     * Deletes a person by the email entered. (assuming that email is unique)
+     *
+     * @param commandArgs full command args string from the user
```

Comment:
Not very descriptive. Better to state what the command string should contain, or just leave out if it's obvious from looking at the method.